lib/exq_ui_web/router.ex

```diff
@@ -29,7 +29,8 @@ defmodule ExqUIWeb.Router do
       options[:live_session_name] || :exq_ui,
       [
         session: {__MODULE__, :__session__, []},
-        root_layout: {ExqUIWeb.LayoutView, :root}
+        root_layout: {ExqUIWeb.LayoutView, :root},
+        on_mount: options[:live_session_on_mount] || __MODULE__
```

Comment:
The on_mount option spec has changed in 0.17. I am assuming you are testing with 0.16. What about only passing this value if the option is set, otherwise don't include the key.

I have pushed some changes to master, please rebase as well